server/schedule/operator/builder.go

```diff
@@ -632,9 +654,9 @@ func (b *Builder) execChangePeerV2(needEnter bool, needTransferLeader bool) {
 
 var stateFilter = filter.StoreStateFilter{ActionScope: "operator-builder", TransferLeader: true}
 
-// check if the peer has the ability to become a leader.
-func (b *Builder) hasAbilityLeader(peer *metapb.Peer) bool {
-	// these roles are not allowed to become leaders.
+// check if the peer is allowed to become the leader.
+func (b *Builder) allowLeader(peer *metapb.Peer, ignoreClusterLimit bool) bool {
```

Comment:
Is there a better name for `ignoreClusterLimit`?

Reply:
I think there are better names but I don't know one.

Reply:
The current name is ok to me.